format-common/src/main/java/io/cdap/plugin/format/plugin/AbstractFileSink.java

```diff
@@ -55,23 +58,31 @@ protected AbstractFileSink(T config) {
 
   @Override
   public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
-    config.validate();
-    FileFormat format = config.getFormat();
-    OutputFormatProvider outputFormatProvider =
-      pipelineConfigurer.usePlugin("outputformat", format.name().toLowerCase(),
-                                   FORMAT_PLUGIN_ID, config.getProperties());
-    if (outputFormatProvider == null) {
-      throw new IllegalArgumentException(String.format("Could not find the '%s' output format plugin.",
-                                                       format.name().toLowerCase()));
+    FailureCollector collector = pipelineConfigurer.getStageConfigurer().getFailureCollector();
+    config.validate(collector);
+
+    FileFormat format = getFileFormat();
+    if (format != null) {
+      ValidatingOutputFormatProvider outputFormatProvider =
```

Comment:
wasn't this renamed to ValidatingOutputFormat?

Reply:
sorry, missed to change the interface name as well: https://github.com/cdapio/cdap/pull/11640